Dshop/apps/products_catalogue/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 4.2.3 on 2023-08-06 17:58
+# Generated by Django 4.2.3 on 2023-08-23 12:47
```

Comment:
What happened here?
All migrations are stored also in DB. By removing and reworking those migrations everyone else will have to do an additional job to catch up. 
Please revert those migrations changes.

Reply:
Django can squash migrations, if you are interested. https://docs.djangoproject.com/en/4.2/topics/migrations/#migration-squashing

Reply:
migrations reverted :)